File: Arrays & Hashing/Contains Duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        Dictionary<int,int> freq = new Dictionary<int,int>();
        for (int i = 0; i < nums.Length; i++)
        {
            if (!freq.ContainsKey(nums[i])) freq.Add(nums[i], 1);
            else return true;
        }
        
        return false;
    }
}

File: Arrays & Hashing/Contains Duplicate.ts
function containsDuplicate(nums: number[]): boolean {
    let charCount = {};

    for (let i: number = 0; i < nums.length; i++) {
        charCount.hasOwnProperty(nums[i])?charCount[nums[i]]++:charCount[nums[i]] = 1;
        if (charCount[nums[i]] > 1) return true;
    }

    return false;
};
